config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -805,3 +642,7 @@ presubmits:
         hostPath:
           path: /sys/fs/cgroup
           type: Directory
+    annotations:
```

Comment:
Is it necessary to set those annotations for presubmit jobs?

Reply:
@fabriziopandini yes, otherwise the presubmits will not have a dashboard and this PR cannot pass CI.

Reply:
got it